### DISCUSSION TITLE
Desktop screenshot

### DISCUSSION THREAD

**Discussion by leschek** [devemux86/cruiser#645] [Q&A] [answered]
I would like to make screenshot of map to show it my friends, but there is very small part of map I would like to include missing. When I zoom out with mouse wheel by just one click, it zooms out too far. Is there way to make zoom more fine, or to make rectangle around area I would like to see on the map?

**Comment by devemux86** [comment]
You can also zoom in / out using the `+` and `-` keyboard keys.

Mouse wheel zooms in / out 1 zoom level at a time. The zoom levels are fixed.
Or your mouse wheel does multiple events at a time, you can check the mouse settings.

**Reply by leschek** [reply]
Thank you for quick answer. It looks it zooms the same way with mouse wheel and with +- keys (by 1 zoom level at a time as you wrote). It's pity, because I would like to include only about 1 cm of map on the top. OK, if there is not fine zooming, is in cruiser possible to go to full screen mode?

**Reply by devemux86** [reply, marked as answer]
You can maximize the window and use the menu "Tools | Screenshot" to take a screenshot of the map and overlays.

**Reply by leschek** [reply]
Unfortunately I'm on max. height of window. I tried to move the cruiser window down (out of screen) and enlarge it on the top (to make it higher then screen), but it doesn't work. 
I guess I will do two screenshots and stitch them together in picture editor.
Thank you for great support.
